hpi-schul-cloud/schulcloud-client
# BC-6460 Remove redis-node package

# Description

QA note:
Change tested on env :
https://bc-6460-one-redis-library.nbc.dbildungscloud.dev/
No issues found. Tests were made in general to check basic functionality of  system  like creating courses plus content, team, homework, adding accounts, learning store , help section , management.

As ioredis library was introduced to project, there was a need to remove redis (redis-node) library


## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6460
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4890


## Changes


## Data Security


## Deployment


## New Repos, NPM packages or vendor scripts


## Screenshots of UI changes


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
